qiita_ware/commands.py

```diff
@@ -120,6 +120,11 @@ def list_remote(URL, private_key):
     ssh = _ssh_session(p_url, private_key)
     valid_files = _list_valid_files(ssh, directory)
     ssh.close()
+
+    # for security, remove key
+    if exists(private_key):
+        remove(private_key)
```

Comment:
Should these two lines be in an `except` clause to guarantee that if something fails the key is removed? Probably a blank `except` just to ensure that no matter what we delete the file.

Reply:
Do you mean 120-122?